k8s/job.go

```diff
@@ -14,6 +14,40 @@ import (
 	yamlutil "k8s.io/apimachinery/pkg/util/yaml"
 )
 
+func ApplyJobExecuterServiceAccount(kubeConfigPath string) error {
+	jobExecuterServiceAccount := `
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: rke-job-executer
+  namespace: kube-system
+`
+
+	jobExecuterClusterRoleBinding := `
```

Comment:
this should be included as cosntants since they are static and will not be changed

Reply:
also i think this is a duplicate